README.md

```diff
@@ -149,7 +149,7 @@ Example command:
 mvn clean verify -Ptestsuite -Dtest=SpecialCharsTest#diacriticsNative \
     -Dquarkus.version=1.6.0.Final -Dquarkus.platform.version=1.6.0.Final \
     -Dquarkus.native.container-build=true -Dquarkus.native.container-runtime=docker\
-    -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:22.2.0-java11
+    -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-graalvmce-builder-image:22.3-java17
```

Comment:
Mandrel one should be used, it's default now

Reply:
Fixed, thank you